internal/controller/nnf_clientmount_controller.go

```diff
@@ -245,6 +264,20 @@ func (r *NnfClientMountReconciler) changeMount(ctx context.Context, clientMount
 		if unmounted {
 			log.Info("Unmounted file system", "Mount path", clientMountInfo.MountPath)
 		}
+
+		// If this is an unmount on a compute node, we can fully deactivate the block device since we won't use it
+		// again. If this is a rabbit node, we do a minimal deactivation. For LVM this means leaving the lockspace up
+		fullDeactivate := false
+		if r.ClientType == ClientCompute {
+			fullDeactivate = true
+		}
+		deactivated, err := blockDevice.Deactivate(ctx, fullDeactivate)
```

Comment:
This is the only place I've found where a full deactivate can happen, and only on the compute node. Did I miss any other cases?

Reply:
Yep, that's it. The VG lockspace stays open on the Rabbit until the VG is destroyed.